commons/crypto.go

```diff
@@ -55,7 +56,7 @@ func ReadCryptoKey(filePath string) ([]byte, error) {
 	}
 
 	if !exists {
-		return nil, fmt.Errorf("Your bazooka config contains secured data but the keyfile can not be found at %s, reason is: %v\n", filePath, err)
+		log.Printf("Your bazooka keyfile can not be found at %s. If you have secured data in your .bazooka.yml, decryption will certainly fail\n", filePath)
```

Comment:
Still, the function should quit since it can't locate the file , and the code below shouldn't be executed.